.github/workflows/linting.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9", "3.10"]
```

Comment:
Why did we take of 3.9 support?

Reply:
I didn't think it was relevant for specifically the linting - I let it continue to run with both 3.9/3.10 for the actual tests, but I don't think version matters for formatting. I am happy to add it back for consistency, though